### ISSUE TITLE
Step 4.70 Calculate Age - Adult/Child for the 3 Year DQ Period

### ISSUE THREAD

**Issue by TanyaPieterse** [abtassociates/LSASampleCode#117]
Step 4.70 of Specs indicate to use EntryDate to calculate client's Status (Adult, Child or Unknown), but maybe we should calculate Status/Age similar to the way AgeGroup is calculated for the active_Enrollment by looking at the EntryDate or First Day of the relevant period (ReportStart).  

In other words for the 3 year DQ Period use EntryDate or dateadd(yy, -2, ReportStart); whichever is later.

This will remove a few false positives in Flag ID 217, where LengthOfStay1 cannot be greater than LengthOfStay3.

Thank you!

**Comment by MollyMcEvilley** [comment]
TL;DR - There is a discrepancy in section 4.70 of the specs which I'm resolving in the sample code in favor of avoiding the flags.  

![image](https://user-images.githubusercontent.com/6156815/57321648-cbb9bf80-70cf-11e9-905b-11f8ed897273.png)

The narrative description of the logic for identifying adults explicitly says that it's based on _DOB_ and _DOBDataQuality_ in relation to _EntryDate_ (only).  

However, the table of conditions below the narrative references both _EntryDate_ and **AgeDate** -- but **AgeDate** isn't defined and could be assumed, based on the narrative, to be a synonym for _EntryDate_.  As such, vendors _could_ leave this alone and have CoCs explain the false positives when they come up.

On the other hand, determining age based on the later of _EntryDate_ and the start of the relevant period is standard in the glossary and in other sections of the LSA specs, so it's not too much of a stretch to update the sample code to clear the flags / resolve the specs discrepancy in favor of the table of conditions instead of the narrative.
